test/e2e-tests.sh

```diff
@@ -264,6 +268,9 @@ function install_consolidated_channel_crds() {
   cp "${CONSOLIDATED_TEMPLATE_DIR}/"*yaml "${KAFKA_CRD_CONFIG_DIR}"
   sed -i "s/REPLACE_WITH_CLUSTER_URL/${KAFKA_CLUSTER_URL}/" ${KAFKA_CRD_CONFIG_DIR}/${KAFKA_CRD_CONFIG_TEMPLATE}
   ko apply -f "${KAFKA_CRD_CONFIG_DIR}" || return 1
+
+  scale_controlplane kafka-ch-controller kafka-ch-dispatcher kafka-webhook
```

Comment:
We should scale `eventing-webhook` and `eventing-controller` as well, `chaosduck` kills pods based on leases present.